### ISSUE TITLE
docker not working

### ISSUE THREAD

**Issue by mackyzap03** [pi-node/instructions#122]
docker says " no container running. what should i do pls help

**Comment by ihatejam** [comment]
Have you done the port test in the Pi app? Doing so should push the "stellar-dummy" container to your pc and then start it up.

**Comment by pilgrim103** [comment]
Firewall issue.  Ports working?  (Use all tests that have been posted everywhere).  Have you added Docker and its programs as Allowed Apps on Firewall?

**Comment by mackyzap03** [comment]
yes i do port test on the app but still not working. im stuck at the last
phase

**Comment by pilgrim103** [comment]
Are you saying that going into the Windows Cmd window, and doing the "netstat -aon" command the ports 31400-31409 have a status of "LISTENING"?

**Comment by ihatejam** [comment]
@mackyzap03 

In docker desktop, do you see a container called "stellar-dummy"? 

If you don't have it, (or if you have it, but it is not in a running state) then a netstat or port check will not yield any results. The action of starting the portchecker inside the node app, has these distinct stages as far as i can tell.
1) Checks if Docker is "ready".
2) Checks to see if the "stellar-dummy" container exists and if not, pushes it down to Docker.
3) Starts the "stellar-dummy" container which then listens on ports 31400-31409 while it is in the running state.

After point number 3), and after the stellar-dummy is shown as running, then, as pilgrim103 mentioned, you can execute "netstat -ano" in a cmd or powershell window. It should show you that those ports are being "listened on".

This does not mean that the ports have been forwarded from your public/external ip address to the nodes private/internal ip address.

Before moving further, please confirm if the stellar-dummy container is present and in the running state.

Also, please forward a screenshot of your troubleshooting page in the Node app.
